### PR TITLE
Improved post input box and added event and project icons.

### PR DESCRIPTION
**Pull request by rak-shit**
Screenshot 2020-02-22 at 11 28 07 AM

### CODE REVIEW

**Comment by rak-shit** [comment]
Screenshot 2020-02-22 at 4 50 18 PM

After Changes

**Comment by rak-shit** [comment]
Screenshot 2020-02-22 at 10 33 41 PM

**Comment by vaibhavdaren** [comment]
@devesh-verma are you writing common scss file ?

File: src/user/dashboard/news-feed/news-feed.js [hunk 1]
```diff
@@ -1,19 +1,165 @@
...
+    
+    handleShow = () => {
+      this.setState({
+        show: true
+      })
+    }
+
+
+render() {
+  const titleStyle = {
```
Comment (vaibhavdaren) [inline src/user/dashboard/news-feed/news-feed.js]: discussed

**Comment by vaibhavdaren** [comment]
Looks Good. Have you tried resizing - Do media queries work?

**Comment by rak-shit** [comment]
> Looks Good. Have you tried resizing - Do media queries work?

Yes they work fine. The page is responsive.

**Comment by vaibhavdaren** [comment]
:) good work

**Comment by rak-shit** [comment]
> :) good work

Sorry, was busy. Thank you btw :)

**Pull request merged**